### PR TITLE
docs: Adds Kinsta Static Site Hosting

### PR DESCRIPTION
**Pull request by palmiak**
## Summary
Adds Kinsta Static Site Hosting deployment guide

## Checklist

- [ ] I have added changeset via `pnpm run change`.
- [x] I have updated the documentation.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: packages/document/docs/en/guide/basic/deploy.mdx [hunk 1]
```diff
@@ -162,3 +162,23 @@ Then click the `Deploy site` button to complete the deployment.
 #### Configure Custom Domain
 
 If you want to bind your site to your own domain, you can configure it in the `Domain management` column of `Netlify`. The specific configuration method can refer to the [Netlify official documentation](https://docs.netlify.com/domains-https/custom-domains/).
+
+### Deploy to Kinsta Static Site Hosting
+
+You can deploy your Vite app as a Static Site on [Kinsta](https://kinsta.com/static-site-hosting/).
```
Comment (chenjiahan) [inline packages/document/docs/en/guide/basic/deploy.mdx:168]: Vite -> Rspress 😄

Comment (palmiak) [reply packages/document/docs/en/guide/basic/deploy.mdx]: 🤦 fixed

File: packages/document/docs/en/guide/basic/deploy.mdx [hunk 2]
```diff
@@ -162,3 +162,23 @@ Then click the `Deploy site` button to complete the deployment.
 #### Configure Custom Domain
 
 If you want to bind your site to your own domain, you can configure it in the `Domain management` column of `Netlify`. The specific configuration method can refer to the [Netlify official documentation](https://docs.netlify.com/domains-https/custom-domains/).
+
+### Deploy to Kinsta Static Site Hosting
+
+You can deploy your Rspress on [Kinsta](https://kinsta.com/static-site-hosting/).
```
Comment (sanyuan0704) [inline packages/document/docs/en/guide/basic/deploy.mdx:168]: `your Rspress site`

**Comment by sanyuan0704** [comment]
Please add chinese doc

**Comment by chenjiahan** [comment]
> Please add Chinese doc

Chinese doc is not required, we can merge this PR first, then create a good-first issue to translate this document, I think Chinese devs will be willing to help this.

**Pull request merged**